pkg/cli/demo.go

```diff
@@ -1029,3 +997,90 @@ func runDemo(cmd *cobra.Command, gen workload.Generator) (err error) {
 
 	return runClient(cmd, conn)
 }
+
+// getServerURL generates a connection URL for the given server.
+func (c *transientCluster) getServerURL(
```

Comment:
This file is getting quite large -- i'm wondering if it is worth to split the demo cluster itself into a separate file.